liquibase-couchbase/src/main/java/liquibase/ext/couchbase/lockservice/LockStatus.java

```diff
@@ -0,0 +1,8 @@
+package liquibase.ext.couchbase.lockservice;
+
+public enum LockStatus {
+    LOCKED_BY_SERVICE,
+    LOCKED_BY_ANOTHER_SERVICE,
+    NO_LOCK
+
```

Comment:
Actually that much better, but i would offer small name improvement according to java convention lock names
```suggestion
public enum LockStatus {
    FREE,
    OWNER,
    NOT_OWNER
    }
```

Reply:
Agree. Will change it

Reply:
Done